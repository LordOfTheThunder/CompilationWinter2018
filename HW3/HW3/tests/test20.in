int potato() {
	int t = 13;
}
void tomato() {
	bool x = potato();
}

void main() { print("Max is cool"); }
