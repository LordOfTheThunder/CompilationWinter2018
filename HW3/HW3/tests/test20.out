---end scope---
t INT 1
line 5: type mismatch
